packages/lodestar/package.json

```diff
@@ -36,6 +36,7 @@
     "pretest": "yarn run check-types",
     "test": "yarn test:unit && yarn test:e2e",
     "test:unit": "LODESTAR_PRESET=minimal nyc --cache-dir .nyc_output/.cache -e .ts mocha 'test/unit/**/*.test.ts'",
+    "test:perf": "LODESTAR_PRESET=minimal nyc --cache-dir .nyc_output/.cache -e .ts mocha 'test/perf/**/*.test.ts'",
```

Comment:
Benchmarks must be run with the `benchmark runner`. The `nyc` binary tracks coverage data which makes no sense in a benchmark.

Follow this script https://github.com/ChainSafe/lodestar/blob/86f9cf24b2546c362b874265539b90e3a6db50ec/packages/beacon-state-transition/package.json#L36

Reply:
When I try to run that command (both when I run for `beacon-state-transition` and when i copy it and run it for `lodestar`) I get this error:

```
✖ Error: Must specify a history option
    at Object.getHistoryProvider (/home/seraph/work/ChainSafe/lodestar/node_modules/@dapplion/benchmark/lib/history/index.js:22:11)
    at Object.run (/home/seraph/work/ChainSafe/lodestar/node_modules/@dapplion/benchmark/lib/run.js:36:39)
    at Object.handler (/home/seraph/work/ChainSafe/lodestar/node_modules/@dapplion/benchmark/lib/cli.js:24:30)
    at /home/seraph/work/ChainSafe/lodestar/node_modules/@dapplion/benchmark/node_modules/yargs/build/index.cjs:1:8973
    at j (/home/seraph/work/ChainSafe/lodestar/node_modules/@dapplion/benchmark/node_modules/yargs/build/index.cjs:1:4852)
    at _.applyMiddlewareAndGetResult (/home/seraph/work/ChainSafe/lodestar/node_modules/@dapplion/benchmark/node_modules/yargs/build/index.cjs:1:8942)
    at _.runCommand (/home/seraph/work/ChainSafe/lodestar/node_modules/@dapplion/benchmark/node_modules/yargs/build/index.cjs:1:7127)
    at Bt.[runYargsParserAndExecuteCommands] (/home/seraph/work/ChainSafe/lodestar/node_modules/@dapplion/benchmark/node_modules/yargs/build/index.cjs:1:52753)
    at Bt.parse (/home/seraph/work/ChainSafe/lodestar/node_modules/@dapplion/benchmark/node_modules/yargs/build/index.cjs:1:36100)
    at Object.<anonymous> (/home/seraph/work/ChainSafe/lodestar/node_modules/@dapplion/benchmark/lib/cli.js:59:6)
```

Do you know what's up with that?

Reply:
Yes, you must provide a history option to tell the tool where to get benchmark data from. Locally you should use `--local` to persist it in the local filesystem

Reply:
I keep getting this error locally as well as in CI when I try to run `yarn run benchmark`

```
Connected to historyProvider: LocalHistoryProvider, dirpath ./benchmark_data
No previous bencharmk found for latestCommitInBranch 'master'
 ✖ TypeError: Cannot read property 'fromBytes' of undefined
    at syncPubkeys (/home/seraph/work/ChainSafe/lodestar/packages/beacon-state-transition/src/allForks/util/epochContext.ts:172:37)
    at createEpochContext (/home/seraph/work/ChainSafe/lodestar/packages/beacon-state-transition/src/allForks/util/epochContext.ts:100:5)
    at Object.createCachedBeaconState (/home/seraph/work/ChainSafe/lodestar/packages/beacon-state-transition/src/allForks/util/cachedBeaconState.ts:70:20)
    at Object.generateCachedState (/home/seraph/work/ChainSafe/lodestar/packages/lodestar/test/utils/state.ts:149:10)
    at Suite.<anonymous> (/home/seraph/work/ChainSafe/lodestar/packages/lodestar/test/perf/chain/stateCache/stateContextCheckpointsCache.test.ts:15:17)
    at Object.create (/home/seraph/work/ChainSafe/lodestar/node_modules/mocha/lib/interfaces/common.js:148:19)
    at context.describe.context.context (/home/seraph/work/ChainSafe/lodestar/node_modules/mocha/lib/interfaces/bdd.js:42:27)
    at Object.<anonymous> (/home/seraph/work/ChainSafe/lodestar/packages/lodestar/test/perf/chain/stateCache/stateContextCheckpointsCache.test.ts:7:1)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Module.m._compile (/home/seraph/work/ChainSafe/lodestar/node_modules/ts-node/src/index.ts:1056:23)
 ```

Reply:
nvm, think i got it.  will push up once resolved